src/bridge/settings/openedx/version_matrix.py

```diff
@@ -57,16 +57,6 @@ def get_item(cls, key: OpenEdxDeploymentName) -> DeploymentEnvRelease:
         return deployment.value
 
 
-default_branding_overrides = {
-    "@edx/frontend-component-footer@npm": (
-        "@mitodl/frontend-component-footer-mitol@latest"
-    ),
-    "@edx/frontend-component-header@npm": (
```

Comment:
We do not have a slot for the footer right now. Also, it is a work in progress by open edx. We should keep the footer override and remove the header override as that will be managed by the slots.